src/me/StevenLawson/TotalFreedomMod/Commands/Command_tag.java

```diff
@@ -17,7 +17,7 @@
 {
     public static final List<String> FORBIDDEN_WORDS = Arrays.asList(new String[]
     {
-        "admin", "owner", "moderator", "developer", "console"
+        "admin", "owner", "moderator", "developer", "console", "sa", "sra", "sta", "0wner", "admim", "server", "founder", "dev"
```

Comment:
Yes I did mean to do that because OPs make there Tag [Admim] and it looks the same